packages/docs/src/components/layout/Layout.tsx

```diff
@@ -94,23 +94,25 @@ const Layout = ({
           className="ds-l-md-col ds-u-padding--0 ds-u-padding-bottom--4 ds-u-padding-top--2 page-main"
         >
           {pageHeader ? pageHeader : <PageHeader frontmatter={frontmatter} theme={theme} />}
-          <article className="ds-u-md-display--flex ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-sm-padding-top--1 ds-u-padding-bottom--3 page-content">
-            <div className="page-content__content ds-l-lg-col--9 ds-u-padding-left--0">
-              <div className="ds-u-display--block ds-u-lg-display--none">
-                <TableOfContentsMobile
+          <article className="ds-u-md-display--flex ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-sm-padding-top--1 ds-u-padding-bottom--3 page-body">
+            <div className="ds-l-row">
+              <div className="ds-l-lg-col--9">
+                <div className="ds-u-display--block ds-u-lg-display--none ds-u-margin-bottom--3">
+                  <TableOfContentsMobile
+                    title={frontmatter.title}
+                    items={tableOfContentsData || []}
+                    slug={slug}
+                  />
+                </div>
+                <div className="page-body__content">{children}</div>
```

Comment:
Removing that one rule for zeroing out padding that used the `.page-content__content` selector allowed me to move this content-wrapping class to be only around the content and not around the mobile table of contents menu. That allowed me to simplify the rules targeting headings.